fix: updating a11y success criteria (#27683)
mdn/content: files/en-us/web/accessibility/understanding_wcag/understandable/index.md

        input situations, not just ones involving sensitive data.
      </td>
      <td>Again, see 3.3.4.</td>
      <tr>

>       <td> 3.3.7 Redundant Entry (A) </td>

Reviewer: ```suggestion
      <td> 3.3.7 Redundant entry (A) </td>
```

      <td>

>       Required information that is previously entered or provided by the user in the same process is either auto-populated or available for the user to choose.

Reviewer: ```suggestion
      Information that is required that was previously entered or provided by the user in the same process or user flow is either auto-populated or made selectable to the user from a list of options, unless re-entering the information is essential or required for security reasons, or if the information is no longer valid.
```

      </td>

>       <td> <strong> Note:</strong> This criteria is not applicable for certain situations. Check out <a href="https://www.w3.org/WAI/standards-guidelines/wcag/new-in-22/#337-redundant-entry-a">the note in WCAG 3.3.7</a> to learn more.</td>

Reviewer: ```suggestion
      <td>Check out <a href="https://www.w3.org/WAI/WCAG22/Understanding/redundant-entry">Understanding redundant entry</a> to learn more.</td>
```

      </tr>
      <tr>

>       <td> 3.3.8 Accessible Authentication (Minimum) (AA)

Reviewer: ```suggestion
      <td> 3.3.8 Accessible authentication (Minimum) (AA)
```

      </td>
      <td>

> States that cognitive function tests like remembering a password is not required for any step in an authentication process unless said provides an alternative, object recognition/personal content(e.g., images, videos, and audio), or mechanicism(e.g., copy and paste and autosave passwords).

Reviewer: ```suggestion
Cognitive function tests, like remembering a password, are not required for any step in an authentication process unless an alternative is provided, such as an object or personal content (e.g., images, videos, and audio) recognition, or a mechanism to assist (e.g., copy and paste and autosave passwords).
```

      </td>

>       <td> Check out <a href="https://www.w3.org/WAI/WCAG22/Understanding/accessible-authentication"> the full documentation</a> for this standard to learn more. </td>

Reviewer: ```suggestion
      <td> Check out the <a href="https://www.w3.org/WAI/WCAG22/Understanding/accessible-authentication"> accessible authentication documentation</a> for this standard to learn more. </td>
```

    </tr>
    <tr>

>     <td> 3.3.9 Accessible Authentication (Enhanced) (AAA) </td>
>     <td>
> Cognitive function tests like remembering a password is not required for any step in an authentication process unless that step either provides an alternative or mechanicism.
>     </td>
> <td> Check out <a href="https://www.w3.org/WAI/WCAG22/Understanding/accessible-authentication-enhanced"> the full documentation</a> for this standard to learn more.</td>

Reviewer: ```suggestion
 <td> 3.3.9 Accessible Authentication (Enhanced) (AAA) </td>
    <td>
A cognitive function test, like remembering a password, must not be required for any step in an authentication process without providing an alternative that does not rely on a cognitive function test or provides a mechanism to assist the user in completing the cognitive function test. Authentication tests that require the user to recognize objects or identify non-text content the user provided to the website are allowed.
    </td>
<td> Check out the <a href="https://www.w3.org/WAI/WCAG22/Understanding/accessible-authentication-enhanced">enhanced accessible authentication documentation (AAA)</a> to learn more.</td>
```
